dynawo/sources/Modeler/DataInterface/IIDM/test/TestStateVariable.cpp

```diff
@@ -329,7 +334,7 @@ TEST(ModelsModelNetwork, TestNetworkCreation) {
     ASSERT_EQ(staticVarCompensators.size(), 1);
     for (std::size_t j = 0, jEnd = staticVarCompensators.size(); j < jEnd; ++j) {
       shared_ptr<StaticVarCompensatorInterface> staticVarCompensator = staticVarCompensators[j];
-      ASSERT_EQ(staticVarCompensator->getRegulationMode(), StaticVarCompensatorInterface::RUNNING_V);
+      ASSERT_EQ(staticVarCompensator->getRegulationMode(), StaticVarCompensatorInterface::STANDBY);
```

Comment:
Would be nice to add a test without the standByAutomaton

Reply:
done!